ui-tests/__test__/main.spec.ts

```diff
@@ -51,4 +51,16 @@ describe('Open MynahUI', () => {
   it('should open a new the tab', async () => {
     await openNewTab(page);
   });
+
+  it('should close the tab with middle click', async () => {
+    await closeTab(page, true, true);
+  });
+
+  it('should open a new tab with double click', async () => {
+    await openNewTab(page, true, true);
+  });
+
+  it('should keep the content inside window boundaries', async () => {
```

Comment:
can we add test for when it correctly evaluate if contents is outside of boundary as well?

Reply:
Currently it already evaluates the boundaries.